components/utilities/dialog/portal.jsx

```diff
@@ -38,24 +40,29 @@ class Portal extends Component {
 		if (typeof this.props.renderTo === 'string') {
 			element = document.querySelector(this.props.renderTo);
 		} else {
-			element = this.props.renderTo || document.body;
+			element = this.props.renderTo || (documentDefined && document.body);
 		}
 		return element;
 	}
 
 	setupPortalNode() {
 		const parentParentNode = this.getPortalParentNode();
 
-		this.portalNode = document.createElement(this.props.renderTag);
-		this.portalNode.setAttribute(
-			'style',
-			'display: block; height: 0px; width: 0px;'
-		);
-		this.portalNode.setAttribute('className', 'design-system-react-portal');
-		parentParentNode.appendChild(this.portalNode);
-		this.portalNodeInstance = this.props.onMount
-			? this.props.onMount(undefined, { portal: this.portalNode })
-			: this.portalNode;
+		this.portalNode = documentDefined
```

Comment:
(minor) This could be `this.portalNode = {};` and the logic moved inside block https://github.com/salesforce/design-system-react/blob/bf5c9c417adb9690a82e0ff1c04fd436c9e7028f/components/utilities/dialog/portal.jsx#L55

Reply:
Good call. I like that it reads easily.